Makefile

```diff
@@ -138,3 +138,7 @@ reload-prod:
 
 stop-prod:
 	@kill -SIGTERM `cat $PROD_PID`
+
+.PHONY: environ
```

Comment:
I made it phony in order to be able to regenerate environ file whenever I wish to